app/scripts/background.js

```diff
@@ -100,6 +117,17 @@ function setupController (initState) {
     return versionedData
   }
 
+  function syncDataWithExtension(state) {
+    if (localStore.isSupported) {
+      try {
+        localStore.set(state)
+      } catch (err) {
+        log.error('error setting state in local store:', err)
```

Comment:
i dont think these get submitted to sentry (but maybe they do)

Reply:
What does get submitted for sure?

Reply:
(we should document our correct logging practice)

Reply:
im not certain atm, but you can test it with the `Test-Metamask` sentry project on dev